index.html

```diff
@@ -149,42 +141,24 @@ <h2 class="Tonic"> Multi-Post <br>Stories</h2>
     <img class="Counter" src="image/Counter7.png" alt="">
     <li><a href="#" class="A">2015</a> </li>
   </ul>
-
-
-
-  <p class="Main-Text5">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
-
- <div class="Languages5">
+   <p class="Main-Text5">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+   <div class="Languages5">
       <button type="submit" value="Submit" class="lan51">html</button>
       <button type="submit" value="Submit" class="lan52">css</button>
       <button type="submit" value="Submit" class="lan53">javascript</button>
   </div>
 
-
   <div class="See-Project5">
     <button type="submit" value="Submit" class="btn3">See Project</button>
   </div>
 
 </div>
 
-
-
-
-
-
-
-    
-
-
- 
-
 <section id="AbtContainter">
- 
-  <div class="Top-Sec">
-   <h2 class="About" >About Myself </h2>
-   <p class="Text" >Hello I’m a software developer! I can  <br> help you build a product , feature or  <br> website Look through some of my <br> work and experience! If you like what <br> you see and have a project you need <br> coded, don’t hestiate to contact me.</p>
-
-    <a href="#" class="Connect1">LET’S CONNECT</a>
+ <div class="Top-Sec">
+    <h2 class="About" >About Myself </h2>
+     <p class="Text" >Hello I’m a software developer! I can  <br> help you build a product , feature or  <br> website Look through some of my <br> work and experience! If you like what <br> you see and have a project you need <br> coded, don’t hestiate to contact me.</p>
+     <a href="#" class="Connect1">LET’S CONNECT</a>
 
 <ul class="Social1">
   <li><a href="#"><img src="image/Vector5.png" alt=""> </li>
```

Comment:
- [ ] Your images are missing an `alt attribute`. :thinking: Currently you left them empty, following the best practices, all images are meant to have an alt tag. **The purpose of alt text is to improve accessibility by describing what an image is showing to visitors who do not have the ability to see them. However, it also helps search engine crawlers and so improves SEO.** :smile: :+1:  I'll suggest you go through your code and add texts in the alt attribute for all the images